topojson/utils.py

```diff
@@ -385,6 +385,8 @@ def serialize_as_geodataframe(fc, crs=None):
     import geopandas
 
     gdf = geopandas.GeoDataFrame().from_features(features=fc["features"], crs=crs)
+    gdf = gdf.set_index(geopandas.pd.json_normalize(fc["features"])["id"].values)
```

Comment:
It's a little weird for me to use like `GeoDataFrame().from_features` and `geopandas.pd.json_normalize`.
`from_features` is a class method, we can directly call it from GeoDataFrame.
`geopandas.pd` is a shortcut to hook pandas, but it's a little bit confusing.

```python
from geopandas import GeoDataFrame
from pandas import json_normalize

return (
    GeoDataFrame.from_features(features=fc["features"], crs=crs)
    .set_axis(json_normalize(fc["features"])["id"].values)
)
```